src/server/pingserver-rs/src/alloc.rs

```diff
@@ -28,15 +29,27 @@ macro_rules! c_str {
 /// Allocator using cc_alloc and cc_free to track and log allocations
 pub struct LoggedAlloc;
 
+const SYS_ALLOC: System = System;
+
 unsafe impl GlobalAlloc for LoggedAlloc {
     unsafe fn alloc(&self, layout: Layout) -> *mut u8 {
         if layout.align() > 16 {
-            return null_mut();
+            // We can't communicate alignment to the ccommon allocator
```

Comment:
We should have a TODO or issue to track fixing this

Reply:
I've added a TODO below here. I think the proper solution here is to just duplicate the logging that ccommon does in rust for this case but I'd rather address that in a different PR if necessary.